### PR TITLE
Packages renaming

### PR DESCRIPTION
**Pull request by pyby**
End of the hobby stuff, we are now one little piece of Play mobile products.

A lot of changes, but this pull request only contains a packages renaming.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,9 +10,9 @@
         </repository>
     </repositories>
 
-    <groupId>com.example</groupId>
+    <groupId>ch.srgssr.playfff</groupId>
     <artifactId>pfff</artifactId>
```
Comment (StaehliJ) [inline pom.xml]: groupId = ch.srgssr

Comment (StaehliJ) [inline pom.xml]: artifactId = playfff

Comment (pyby) [reply pom.xml]: Fixed.

Comment (pyby) [reply pom.xml]: Fixed.

**Pull request merged**